test-apps/display-performance-test-app/src/frontend/TestRunner.ts

```diff
@@ -1028,51 +1037,50 @@ function getRenderOpts(curRenderOpts: RenderSystem.Options): string {
           }
         }
         break;
-      case "preserveShaderSourceCode":
-        if (value) optString += "+shadeSrc";
-        break;
+      }
       case "displaySolarShadows":
-        if (!value) optString += "-solShd";
+        if (!curRenderOpts[key]) optString += "-solShd";
         break;
-      case "logarithmicZBuffer":
-        if (value) optString += "+logZBuf";
+      case "logarithmicDepthBuffer":
+        if (!curRenderOpts[key]) optString += "-logZBuf";
         break;
       case "useWebGL2":
-        if (value) optString += "+webGL2";
+        if (!curRenderOpts[key]) optString += "-webGL2";
         break;
-      case "antialiasSamples":
-        if (value > 1) optString += `+aa${value as number}`;
+      case "antialiasSamples": {
+        const value = curRenderOpts[key];
+        if (value && value > 1) optString += `+aa${value}`;
         break;
-      default:
```

Comment:
Many options we have added or will add in future should not show up here. If we want them to, we must add them explicitly.